composer.json

```diff
@@ -178,23 +177,23 @@
         "drupal/field_group": "^3.0-rc1",
         "drupal/focal_point": "^1.0",
         "drupal/formblock": "^1.0",
-        "drupal/fullcalendar_view": "^4.5",
+        "drupal/fullcalendar_view": "^5.0.0",
         "drupal/ga_login": "^1.0@alpha",
         "drupal/geocoder": "^3.0",
-        "drupal/geolocation": "3.1.0",
+        "drupal/geolocation": "^3.1.0",
```

Comment:
I know that we cannot use Geolocation 3.2.0 due to a bug during edit mode so let's hope that it jumps to 3.3.0+.

https://judasdg.atlassian.net/browse/TC-288?focusedCommentId=63096

Reply:
Thanks Mel.  Good catch from memory, but I've also added a patch for that issue. https://github.com/JudicialCouncilOfCalifornia/trialcourt/pull/1718/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R365